examples/plot_simulate_alpha.py

```diff
@@ -19,47 +23,87 @@
-# Update a few of the default parameters related to visualisation
-params.update({
-    'dipole_scalefctr': 150000.0,
-    'dipole_smooth_win': 0,
-    'tstop': 310.0,
-})
-
 ###############################################################################
 # Now let's simulate the dipole and plot it. To excite the network, we add a
 # ~10 Hz "bursty" drive starting at 50 ms and continuing to the end of the
 # simulation. Each burst consists of a pair (2) of spikes, spaced 10 ms apart.
 # The occurrence of each burst is jittered by a random, normally distributed
 # amount (20 ms standard deviation). We repeat the burst train 10 times, each
-# time with unique randomization. The drive is only connected to the distal
+# time with unique randomization. The drive is only connected to the proximal
 # (dendritic) AMPA synapses on L2/3 and L5 pyramidal neurons.
+params['tstop'] = 310
 net = Network(params)
 
-weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+location = 'proximal'
+burst_std = 20
+weights_ampa_p = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+syn_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                'L5_basket': 1., 'L5_pyramidal': 1.}
+
 net.add_bursty_drive(
-    'bursty', tstart=50., burst_rate=10, burst_std=20., numspikes=2,
-    spike_isi=10, repeats=10, location='distal', weights_ampa=weights_ampa,
-    seedcore=4)
+    'alpha_prox', tstart=50., burst_rate=10, burst_std=burst_std, numspikes=2,
+    spike_isi=10, repeats=10, location=location, weights_ampa=weights_ampa_p,
+    synaptic_delays=syn_delays_p, seedcore=14)
 
-dpl = simulate_dipole(net)
+dpl = simulate_dipole(net, postproc=False)
 
+###############################################################################
+# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
+# power spectral density. Note that the SciPy-function
+# `~scipy.signal.spectrogram` is used to create the plot. The
+# ``dpl[trial_idx].scale()`` call relates to the amount of cortical tissue
+# necessary to observe the electric current dipole outside the head with M/EEG.
+import matplotlib.pyplot as plt
+from hnn_core.viz import plot_dipole, plot_spectrogram
 trial_idx = 0  # single trial simulated
-dpl[trial_idx].plot()
+dpl[trial_idx].scale(150000)
+fig, axes = plt.subplots(2, 1)
+tmin = 20  # exclude initial burn-in period
+plot_dipole(dpl[trial_idx], tmin=tmin, ax=axes[0], show=False)
+plot_spectrogram(dpl[trial_idx], fmin=0., fmax=40., tmin=tmin, ax=axes[1])
+plt.tight_layout()
+###############################################################################
+# The next step is to add a simultaneous 10 Hz distal drive. Due to the
+# stochasticity of input spike timing, the proximal and distal spikes
+# occasionally arrive at the same time which will result in a beta frequency
+# (15-30 Hz) event. The higher frequency activity arises from biophysical
+# properties intrinsic to the cortical neurons.
```

Comment:
I don't think I quite agree with this, I'll make a general comment later. Please consider revising.

Reply:
Using your suggested description, definitely clearer and to the point.